### PR TITLE
Background scroll error fixed

### PR DESCRIPTION
**Pull request by MadhavBahlMD**
Fixes issue #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber]
#220 

Changes: [Add here what changes were made in this issue and if possible provide links.]
Screenshots for the change:
Earlier:
image

Now:

image

Background position fixed.

### CODE REVIEW

**Comment by MadhavBahlMD** [comment]
@jenkoian  Issue is fixed :)

File: public/css/stars.css [hunk 1]
```diff
@@ -32,13 +34,18 @@
...
     animation: move-stars-two 10s linear infinite;
     -ms-animation: move-stars-two 10s linear infinite;
     -moz-animation: move-stars-two 10s linear infinite;
     -webkit-animation: move-stars-two 10s linear infinite;
 }
 
+#bkg{
+    background-attachment: fixed;
+    position: fixed;
+}
```
Comment (jenkoian) [inline public/css/stars.css]: Don't need this, apply the styles directly on the div with class `fixed` and you're already adding the `background-attachment: fixed` on the stars above.

Comment (MadhavBahlMD) [reply public/css/stars.css]: I will do it right away

**Comment by MadhavBahlMD** [comment]
@jenkoian Conflicts resolved 👍

**Comment by jenkoian** [comment]
Fixed in #222

**Pull request closed without merging**